db/dbformat.h

```diff
@@ -562,18 +562,25 @@ inline uint64_t GetInternalKeySeqno(const Slice& internal_key) {
 //    allocation for smaller keys.
 // 3. It tracks user key or internal key, and allow conversion between them.
 class IterKey {
+  static constexpr char kTsMin[] = "\x00\x00\x00\x00\x00\x00\x00\x00";
```

Comment:
I think it would be nice to add the usual comment here about only 64-bit timestamps being supported currently.